src/contrib/cluster/Akka.Cluster.Sharding.Tests/ShardRegionQueriesSpecs.cs

```diff
@@ -70,11 +70,11 @@ private string ExtractShardId(object message)
             switch (message)
             {
                 case int i:
-                    return (i % 10).ToString();
+                    return (i % 10 + 1).ToString();
                 // must support ShardRegion.StartEntity in order for
                 // GetEntityLocation to work properly
                 case ShardRegion.StartEntity se:
-                    return se.EntityId;
+                    return (int.Parse(se.EntityId) % 10 + 1).ToString();
```

Comment:
Change the test `IMessageExtractor` so that the `ShardId` does not equal to `EntityId` to catch actor path bugs

Reply:
doh - that's a good catch